test-plan.md

```diff
@@ -155,7 +155,9 @@ void Foo(){
 Note that the pipe in the below examples indicates the cursor.
 * Typing `<environment>` prompts you with HTML completion for the `environment` tag and on completion commit auto-completes the closing tag.
 * Type `<cache |`, hit ctrl + space, search for `asp-vary-by-user`, commit the completion. Ensure that it auto-completes the attribute to: `<cache asp-vary-by-user`.
```

Comment:
Not able to get this completion.

https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/built-in/cache-tag-helper?view=aspnetcore-3.1

Is this still a valid property?

Reply:
Looks like not 😄

Reply:
Gotcha, removed it from the test plan 😄